templates/govuk-frontend.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<!--[if lt IE 9]><html class="lte-ie8" lang="en"><![endif]-->
+<!--[if gt IE 8]><!--><html lang="en" class="govuk-template"><!--<![endif]-->
+{% load static %}
+<head>
+  <meta charset="utf-8">
+  <title>
+    {% block title %}
+    {% endblock %}
+  </title>
+  <meta name="viewport" content="width=device-width, initial-scale=1, viewport-fit=cover">
+  {# Ensure that older IE versions always render with the correct rendering engine #}
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="theme-color" content="{{ themeColor | default:'#0b0c0c' }}" />
```

Comment:
Should we get rid of `themeColour`? Presume its a hangover from control panel or is it used?

Reply:
I think it's also used in the UI currently. Might be worth keeping it in?

Reply:
Just cant tell see `themeColor` comes from, so will it always just use the default? Or does it come from the gov uk frontend?

Reply:
By the looks of it, it's always set to the default. Theme colour is also in the govuk site. I don't think it hurts to keep it

Reply:
Didnt mean remove the whole `<meta>` tag, just the use of reading from a context variable `themeColor` that isnt actually set anywhere. But no doesnt hurt to leave it I guess, just a bit confusing to see (and also it uses the wrong case)